src/pages/sighting/encounters/Encounters.jsx

```diff
@@ -171,6 +171,7 @@ export default function Encounters({
       />
 
       <ManuallyAssignModal
+        sightingGuid={sightingId}
```

Comment:
Might be worth it if you're in here anyway to change sightingId to sightingGuid as well...

Reply:
I was gonna but i realized i have multiple PRs interacting with this same file and didnt wanna create a bunch of merge conflicts!